applications/StructuralMechanicsApplication/tests/periodic_test/periodic_linear_test_parameters.json

```diff
@@ -0,0 +1,145 @@
+        "multi_point_constraints_used"       : true,
+        "reform_dofs_at_each_step"           : true,
+        "line_search"                        : false,
+        "convergence_criterion"              : "Residual_criterion",
+        "displacement_relative_tolerance"    : 0.0001,
+        "displacement_absolute_tolerance"    : 1e-9,
+        "residual_relative_tolerance"        : 0.0001,
+        "residual_absolute_tolerance"        : 1e-9,
+        "max_iteration"                      : 10,
+        "linear_solver_settings"             : {
+                "solver_type"                    : "AMGCL",
+                "max_iteration"                  : 500,
+                "tolerance"                      : 1e-6,
+                "provide_coordinates"            : false,
+                "smoother_type"                  : "ilu0",
+                "krylov_type"                    : "cg",
+                "gmres_krylov_space_dimension"   : 100,
+                "use_block_matrices_if_possible" : false,
+                "coarsening_type"                : "aggregation",
+                "scaling"                        : true,
+                "verbosity"                      : 0
+        },
+        "problem_domain_sub_model_part_list" : ["Parts_solid"],
+        "processes_sub_model_part_list"      : ["DISPLACEMENT_fixedAxis","PointLoad3D_forceOuter"],
+        "rotation_dofs"                      : false
+    },
+    "processes" : {
+        "constraints_process_list" : [{
+            "python_module" : "assign_vector_variable_process",
+            "kratos_module" : "KratosMultiphysics",
+            "help"          : "This process fixes the selected components of a given vector variable",
+            "process_name"  : "AssignVectorVariableProcess",
+            "Parameters"    : {
+                "mesh_id"         : 0,
+                "model_part_name" : "DISPLACEMENT_fixedAxis",
+                "variable_name"   : "DISPLACEMENT",
+                "value"           : [0.0,0.0,0.0],
+                "interval"        : [0.0,"End"]
+            }
+        },{
+            "python_module" : "apply_periodic_boundaray_condition_process",
+            "kratos_module" : "KratosMultiphysics",
+            "process_name"  : "ApplyPeriodicBoundaryConditionProcess",
+            "Parameters"    : {
+                "computing_model_part_name"   : "computing_domain",
+                "model_part_name"             : "Structure",
+                "first_model_part_name"       : "GENERIC_horiSurface",
+                "second_model_part_name"      : "GENERIC_vertSurface",
+                "interval"                    : [0.0, 1e30],
+                "variable_names"              : ["DISPLACEMENT"],
+                "center"                      : [0,0,0],
+                "axis_of_rotation"            : [0.0,0.0,1.0],
+                "angle"                       : 90.0
+            }
+        }],
+        "loads_process_list"       : [{
```

Comment:
I am curious, how do you apply the centrifugal force?

Reply:
In this validation test, its not centrifugal force, its just a point load on the outer surface. Is there a possibility to apply a load via a custom function in tests ?